client/components/catalog/Card.vue

```diff
@@ -34,15 +36,14 @@ import first from 'lodash/first';
 import get from 'lodash/get';
 import { getSchema } from 'shared/activities';
 import { mapActions } from 'vuex';
-import truncate from 'truncate';
 
 export default {
   props: {
     repository: { type: Object, required: true }
   },
   computed: {
-    name: ({ repository }) => truncate(repository.name, 70),
-    description: ({ repository }) => truncate(repository.description, 100),
+    name: ({ repository }) => repository.name,
```

Comment:
No longer needed but will ignore it for now since you v-binded repo in one of the following PRs